feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.spec.ts

```diff
@@ -223,8 +227,26 @@ describe('ConfigUtilsService', () => {
   });
 
   describe('createOvGroupId', () => {
+    it('should create a group id from its 1 parameter', () => {
+      expect(classUnderTest.createOvGroupId(undefined, 'B')).toBe('B-ovGroup');
```

Comment:
Strict mode issue. Please remove all strict mode issues in this test class!

Reply:
Done!